deploy/cr/cr_minimal.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: infinispan.org/v1
 kind: Infinispan
 metadata:
-  name: example-infinispan
```

Comment:
Any particular reason to change this name? I'd leave it as is (and also in `cr_with_configmap`). You'd normally run one or the other as you try it out things, so it's fine to have the same example name.

Reply:
I consider these files as example files, mostly for documentation or our tutorials, so I think it's easier to stick just one name here.

Reply:
I can't remember what was the idea... so I guess it's ok to use just one name